app/services/workflows/main_workflow.rb

```diff
@@ -2,34 +2,62 @@ module Workflows
+                                                            partner_capitals_data: partner.capitals_data,
+                                                            date_of_birth: applicant.details.date_of_birth,
+                                                            level_of_help: assessment.level_of_help,
+                                                            submission_date: assessment.submission_date,
+                                                            partner_date_of_birth: partner.details.date_of_birth,
+                                                            receives_asylum_support: applicant.details.receives_asylum_support)
+                               else
+                                 PassportedWorkflow.call(proceeding_types: assessment.proceeding_types,
+                                                         capitals_data: applicant.capitals_data,
+                                                         date_of_birth: applicant.details.date_of_birth,
+                                                         submission_date: assessment.submission_date,
+                                                         level_of_help: assessment.level_of_help,
+                                                         receives_asylum_support: applicant.details.receives_asylum_support)
+                               end
+                             else
+                               result = NonPassportedWorkflow.call(assessment:, applicant:, partner:)
+                               assessment.add_remarks! result.remarks
+                               result.calculation_output
+                             end
+        calculation_output.tap do
+          # we can take the lower threshold from the first eligibility records as they are all the same
+          lower_capital_threshold = calculation_output.capital_subtotals.eligibilities.first.lower_threshold
+          assessed_capital = calculation_output.capital_subtotals.combined_assessed_capital
+
+          remarks = RemarkGenerators::Orchestrator.call(employments: applicant.employments,
+                                                        gross_income_summary: assessment.applicant_gross_income_summary,
+                                                        outgoings: applicant.outgoings,
+                                                        liquid_capital_items: applicant.capitals_data.liquid_capital_items,
+                                                        state_benefits: applicant.state_benefits,
+                                                        lower_capital_threshold:,
+                                                        child_care_bank: calculation_output.applicant_disposable_income_subtotals.child_care_bank,
+                                                        assessed_capital:)
+          assessment.add_remarks!(remarks)
```

Comment:
@starswan We were concating remarks before and made single insert. Now we have 2 inserts for applicant and partner. Would it create any issue?